testing/run-scripts/run_cloud.sh

```diff
@@ -188,4 +188,20 @@ if ! docker ps -a | grep uproxy-sshd >/dev/null; then
   fi
 fi
 
+echo "CLOUD_INSTALL_PROGRESS 95"
+
+# Boot monitoring servers and metrics.
```

Comment:
Let's have instructions somewhere for now since it's basically just us running this.